src/cli.ts

```diff
@@ -294,7 +294,7 @@ async function handleConfigCopy(options: Options) {
 export async function handleTemplate() {
   const cwd = process.cwd();
   const sourceDirName = path.join(__dirname, '../../template');
-  const targetDirName = path.join(cwd, 'src');
+  const targetDirName = cwd;
```

Comment:
Right now the code only checks if there's a `.ts` file inside of `targetDirName` to determine if it should copy the template files or not. Now that `targetDirName` changed to `cwd()` this check will always be false because we don't have any top-level `.ts` files and we would thus potentially override existing code files.